src/lib/components/ui/Toggle/README.mdx

```diff
@@ -192,3 +192,18 @@ interfere with the API, and they will be passed to the `input` HTML element.
 This is useful mainly to improve component's accessibility.
 
 <Props table of={Toggle} />
+
+## Theming
+
+Head to [Forms Theming](/customize/theming/forms) to see shared form theming
+options. On top of that, the following options are available for Toggle.
+
+| Custom Property                                                    | Description                                    |
+|--------------------------------------------------------------------|------------------------------------------------|
+| `--rui-form-field-check-input-toggle-width`                | Input width (height is shared with other check fields) |
```

Comment:
Isn't it possible to make it shorted? Why `check` is in the class?

Reply:
Custom properties naming is a bit (a lot) complicated:

```
- project prefix (rui)
  - global property type (color) | component name (button) | component type (form-field)
    - forms only, optional: form field category (check | box*)
    - all components, optional: variant | priority | feedback state
      - optional: element name
        - optional: element type
          - interaction state
            - property name
```

It's a very approximate illustration but this is about how it works.

From this point of view I should even spread `box` over the box-field-related custom properties (and not delete `check`) to be consistent with what I say here * 🙂.

So it seems I should revisit all form custom properties and either use the convention everywhere — or flatten them all as much as possible without compromising their descriptiveness (box input size vs. check input size etc.).

Reply:
Relates to #163.

Reply:
OK, I'll unify naming here and change the syntax in #243 so it's more readable.